src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -305,6 +305,97 @@ public void ComboBox_BackColor_ShouldSerializeValue_Success()
         Assert.False(property.ShouldSerializeValue(control));
     }
 
+    [WinFormsFact]
+    public void ComboBox_MeasureItem_AddHandler()
+    {
+        using ComboBox control = new();
+        int initialItemHeight = control.ItemHeight;
+
+        MeasureItemEventHandler handler = (sender, e) => { };
+        control.MeasureItem += handler;
+
+        control.ItemHeight.Should().NotBe(initialItemHeight);
+    }
+
+    public class TestComboBox : ComboBox
+    {
+        public void TriggerMeasureItem(MeasureItemEventArgs e)
+        {
+            base.OnMeasureItem(e);
+        }
+    }
+
+    [WinFormsFact]
+    public void ComboBox_MeasureItem_RemoveHandler()
+    {
+        using TestComboBox control = new();
+        int handlerCallCount = 0;
+        MeasureItemEventHandler handler = (sender, e) => { handlerCallCount++; };
+
+        control.MeasureItem += handler;
+
+        // Simulate the MeasureItem event
+        control.TriggerMeasureItem(new MeasureItemEventArgs(Graphics.FromHwnd(IntPtr.Zero), 0, 0));
+
+        control.MeasureItem -= handler;
+
+        // Simulate the MeasureItem event again to ensure the handler was removed
+        control.TriggerMeasureItem(new MeasureItemEventArgs(Graphics.FromHwnd(IntPtr.Zero), 0, 0));
+
+        handlerCallCount.Should().Be(1, "The MeasureItem event handler was not removed as expected.");
+    }
+
+    public class TestableComboBox : ComboBox
```

Comment:
Since this is only being used in this class let's make it private
```suggestion
    private class TestableComboBox : ComboBox
```